.github/ISSUE_TEMPLATE/test-bug.md

```diff
@@ -0,0 +1,31 @@
+---
+name: "🐛 Test Bug Report"
```

Comment:
Should we change to "🧪 Test Code Bug Report" to show it is not Testing the Bug Reporting?